### DISCUSSION TITLE
Need help adding a new section to the cv.md page in my custom Al-folio site

### DISCUSSION THREAD

**Discussion by DamilolaOwolabi** [alshedivat/al-folio#2614] [Q&A] [answered]
Hi, please i need help adding a new section in my cv.md page in my al-folio site. The section is dedicated to all the trainings i have made in my career  I have update some necessary changes that i thought might work.

- I updated the assets/json/resume.json file
- I added the section to my config file.
-  I added a when statement for the _layouts/cv.liquid file
- I added a new liquid file in the includes folder with codes for the new section.

Please is there anything else i might be missing ? As the changes i made is not reflected on the site.

**Comment by george-gca** [comment]
> I added a when statement for the _layouts/cv.liquid file

Where exactly? Be aware that we use `unless` to check for the jsonresume, meaning that the actual part responsible for using the jsonresume is inside the else statement:

https://github.com/alshedivat/al-folio/blob/e46a7941b216c68493158fe412467c1a11fb8b54/_layouts/cv.liquid#L50

**Reply by DamilolaOwolabi** [reply, marked as answer]
Thank you so much for your help George. I was able to figure out the problem. I forgot to update the cv.liquid file to match the changes i made.

`
{% when 'training' %}
                  {% include resume/training.liquid %}
`
